types/src/common_params.rs

```diff
@@ -105,12 +107,15 @@ impl CommonParams {
     pub fn min_custom_transaction_cost(&self) -> u64 {
         self.min_custom_transaction_cost
     }
+    #[deprecated]
     pub fn min_store_transaction_cost(&self) -> u64 {
```

Comment:
It seems that we can remove these functions instead of deprecating since the previous commit removed the related transaction.

Reply:
Here I also hope to remind you that `min_[de]compose_asset_cost` and the likes are not removed either. According to my examination over the codebase, they are not used elsewhere except related to RPC. The store/remove related functions are in the exactly same situation.

Whlie re-examining the relavent codes per your comment, I've found that we may remove these functions you mentioned if I do either of the following:

1. Remove the cost parameters completely from RPC and all the related e2e tests
1. Just break ties of those functions to RPC and remove the functions (simpler)

IMHO, the `min_[de]compose_asset_cost` things need the same treatment. So it was my proposal to treat them algether as a follow-up for this PR. But if you want me to take immediate action where possible, I hope to ask you of which way (1 or 2) to go for it.

Reply:
How about removing the store/remove related features in this commit?
The compose/decompose related features will be removed by https://github.com/CodeChain-io/foundry/pull/11

Reply:
I filed issue #96, but I can improve this PR to cover the issue #96 but only for the store/remove Tx-related codes. Is that what you want? If so, I am happy to do it, assuming the other entries are handled as per #11.

Reply:
Yes, that's what I wanted. Thank you for doing it.